tests/e2e/autoscale/02-install.yaml

```diff
@@ -1,5 +1,6 @@
 # This updates an existing deployment with a new Spec.
 # Target memory utilization is now added. 
+# scaledown behavior is now added.
```

Comment:
`scaleDown` or both `scaleUp` and `scaleDown`?

Reply:
Thanks for the review! I have fixed the comment now